src/Test/PluginTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
```

Comment:
Test namespace should match the regular namespace plus "/Test" so if you have
`CF\API\Plugin` the namespace of its test should be `CF\Test\API\Plugin`